layouts/partials/fragments/contact.html

```diff
@@ -33,71 +33,71 @@ <h2>{{- . | markdownify -}}</h2>
-                {{ with $data.fields.name }}
+                {{ with $contact.fields.name }}
                   <div class="form-group">
                     <input
                       id="name"
                       name="name"
                       class="form-control"
                       type="text"
-                      data-validation-error-msg="{{ .error | default (i18n "contact.defaultNameError") }}"
-                      data-validation-error-msg-container="#name-error"
+                      contact-validation-error-msg="{{ .error | default (i18n "contact.defaultNameError") }}"
+                      contact-validation-error-msg-container="#name-error"
                       placeholder="{{ with .text }}{{ . | markdownify }}{{ end }}"
                       required>
                     <div class="px-2 d-none" id="name-error"></div>
                   </div>
                 {{ end }}
-                {{ with $data.fields.email }}
+                {{ with $contact.fields.email }}
                   <div class="form-group">
                     <input
                       id="email"
                       name="email"
                       class="form-control"
                       type="text"
-                      data-validation="email"
-                      data-validation-error-msg="{{ .error | default (i18n "contact.defaultEmailError") }}"
-                      data-validation-error-msg-container="#email-error"
+                      contact-validation="email"
+                      contact-validation-error-msg="{{ .error | default (i18n "contact.defaultEmailError") }}"
+                      contact-validation-error-msg-container="#email-error"
                       placeholder="{{ with .text }}{{ . | markdownify }}{{ end }}"
                       required>
                     <div class="px-2 d-none" id="email-error"></div>
                   </div>
                 {{ end }}
-                {{ with $data.fields.phone }}
+                {{ with $contact.fields.phone }}
                   <div class="form-group">
                     <input
                       id="phone"
                       name="phone"
                       class="form-control"
                       type="text"
-                      data-validation="custom"
-                      data-validation-regexp="^([0-9,\+]+)$"
-                      data-validation-optional="true"
-                      data-validation-error-msg="{{ .error | default (i18n "contact.defaultPhoneError") }}"
-                      data-validation-error-msg-container="#phone-error"
+                      contact-validation="custom"
+                      contact-validation-regexp="^([0-9,\+]+)$"
+                      contact-validation-optional="true"
+                      contact-validation-error-msg="{{ .error | default (i18n "contact.defaultPhoneError") }}"
+                      contact-validation-error-msg-container="#phone-error"
```

Comment:
This change breaks the contact form. Search and replace did replace a bit too much here ;)